python/rmm/_lib/device_buffer.pyx

```diff
@@ -50,6 +52,14 @@ cdef class DeviceBuffer:
         buf.c_obj = move(ptr)
         return buf
 
+    cpdef bytes tobytes(self):
+        cdef bytes b = PyBytes_FromStringAndSize(NULL, self.c_size())
+        cdef void* p = <void*>PyBytes_AS_STRING(b)
```

Comment:
The CPython API indicates this adds a null byte at the end which is unintended here, no?

Reply:
Where did you see this?

Reply:
Had a brainfart and read the wrong API 😅

Reply:
No worries 🙂